flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/ContinuousFileProcessingRescalingTest.java

```diff
@@ -67,38 +65,47 @@
 	*/
 	@Test
 	public void testReaderScalingDown() throws Exception {
-		HarnessWithFormat[] beforeRescale = buildAndStart(5, 15);
+		HarnessWithFormat[] beforeRescale = {};
```

Comment:
optional nit: use `CloseableRegistry`?

Reply:
Hmm...looks like it swallows Exceptions.
I'd like to see if something wrong happens.

Reply:
I recommend to use Guava's `Closer`, which will close everything and add suppressions if necessary.

Reply:
As I suggested I think using some generic `Closer` would be better, but let's not block this PR on this (optional nit). I will merge it as it is, we can clean up the test later on.